src/game/types/index.ts

```diff
@@ -146,7 +146,15 @@ export interface ITable {
 }
 
 export interface IGame {
+  /**
+   * The IPlayer['id'] of the user who owns the current session.
+   */
+  readonly userPlayerId: IPlayer['id']
```

Comment:
based on your comment i wonder if an alternate name to this would be more easy to understand. some ideas that come to mind are `primaryPlayerId` or `sessionOwnerPlayerId`

Reply:
Good idea. Changed to `sessionOwnerPlayerId` in https://github.com/jeremyckahn/farmhand-shuffle/commit/b1a3b95dea0fd64accbc17113e31c34bcdb0b86c!